lib/studies/reply/adaptive_nav.dart

```diff
@@ -824,7 +834,22 @@ class _ReplyFab extends StatelessWidget {
     } else {
       return FloatingActionButton(
         heroTag: 'Bottom App Bar FAB',
```

Comment:
The FAB should have a tooltip set for a11y. And the value should change depending on if on mail view or not

Reply:
Will add.